corehq/apps/app_manager/management/commands/build_apps.py

```diff
@@ -8,13 +8,6 @@
 import os
 from corehq.apps.app_manager.models import Application, RemoteApp
 
-try:
-    from guppy import hpy
-    track_perf = True
-except ImportError:
-    track_perf = False
```

Comment:
I think better to not modify this file. The command will still work if guppy is not installed. I believe when I checked last guppy did not officially support Python3 (so we should remove it from our requirements files), but I think it can be installed on Python3.